live-examples/js-examples/object/meta.json

```diff
@@ -1,148 +1,179 @@
-            "title": "JavaScript Demo: Object.keys()",
-            "type": "js"
-        },
-        "objectPreventExtensions": {
-            "exampleCode": "./live-examples/js-examples/object/object-preventextensions.html",
-            "fileName": "object-preventextensions.html",
-            "title": "JavaScript Demo: Object.preventExtensions()",
-            "type": "js"
-        },
-        "objectPrototypeHasOwnProperty": {
-            "exampleCode": "./live-examples/js-examples/object/object-prototype-hasownproperty.html",
-            "fileName": "object-prototype-hasownproperty.html",
-            "title": "JavaScript Demo: Object.prototype.hasOwnProperty()",
-            "type": "js"
-        },
-        "objectIsPrototypePrototypeOf": {
-            "exampleCode": "./live-examples/js-examples/object/object-prototype-isprototypeof.html",
-            "fileName": "object-prototype-isprototypeof.html",
-            "title": "JavaScript Demo: Object.prototype.isPrototypeOf()",
-            "type": "js"
-        },
-        "objectPrototypePropertyIsEnumerable": {
-            "exampleCode": "./live-examples/js-examples/object/object-prototype-propertyisenumerable.html",
-            "fileName": "object-prototype-propertyisenumerable.html",
-            "title": "JavaScript Demo: Object.prototype.propertyIsEnumerable()",
-            "type": "js"
-        },
-        "objectPrototypeSeal": {
-            "exampleCode": "./live-examples/js-examples/object/object-prototype-seal.html",
-            "fileName": "object-prototype-seal.html",
-            "title": "JavaScript Demo: Object.prototype.seal()",
-            "type": "js"
-        },
-        "objectPrototypeToLocaleString": {
-            "exampleCode": "./live-examples/js-examples/object/object-prototype-tolocalestring.html",
-            "fileName": "object-prototype-tolocalestring.html",
-            "title": "JavaScript Demo: Object.prototype.tolocalestring()",
-            "type": "js"
-        },
-        "objectPrototypeToString": {
-            "exampleCode": "./live-examples/js-examples/object/object-prototype-tostring.html",
-            "fileName": "object-prototype-tostring.html",
-            "title": "JavaScript Demo: Object.prototype.toString()",
-            "type": "js"
-        },
-        "objectPrototypeValueOf": {
-            "exampleCode": "./live-examples/js-examples/object/object-prototype-valueof.html",
-            "fileName": "object-prototype-valueof.html",
-            "title": "JavaScript Demo: Object.prototype.valueOf()",
-            "type": "js"
-        },
-        "objectValues": {
-            "exampleCode": "./live-examples/js-examples/object/object-values.html",
-            "fileName": "object-values.html",
-            "title": "JavaScript Demo: Object.values()",
-            "type": "js"
-        }
-    }
-}
+{
+    "pages": {
+        "objectAssign": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
```

Comment:
This commit seems to undo the changes that the last commit made to live-examples/js-examples/object/meta.json - so e.g. baseTmpl is reinstated here. Is that intentional?

Reply:
Nope, just need to rebase against master to get the latest changes ;)

Reply:
Hmmm, I did a diff between between the `meta.json` in this branch and the one on master and find no difference. Was `baseTmpl` removed here? Perhaps it is still in a PR that needs to be merged? @wbamberg

Reply:
The first commit in this PR (https://github.com/mdn/interactive-examples/pull/1142/commits/95d94183e7eee55881489623463dd801b80ff0d4) removes `baseTmpl` from all meta.jsons, and adjusts some paths, as far as I can tell (e.g. https://github.com/mdn/interactive-examples/pull/1142/commits/95d94183e7eee55881489623463dd801b80ff0d4#diff-a19506ff399b51802f2918aefbf9d85c).

But the second commit (https://github.com/mdn/interactive-examples/pull/1142/commits/94dbbb194bdf72b5a185214665e38647d6050d90) seems to reinstate `baseTmpl`, and undo the path changes, for live-examples/js-examples/object/meta.json, presumably as a result of the catch-up integration for https://github.com/mdn/interactive-examples/pull/1127 . I assume that's not intentional, so this PR needs to re-remove `baseTmpl` and reapply the path changes for this file?

Reply:
You are indeed correct. Let me have a look at those. It was definitely a copy/paste manual merge thing. Thanks @wbamberg